BUILD

```diff
@@ -4783,6 +4783,26 @@ grpc_cc_library(
     deps = ["gpr"],
 )
 
+grpc_cc_library(
```

Comment:
Let's make two targets here... one for the client transport and one for the server.

(in general, unless there's strong reason not to - we try for one `.h` and one `.cc` per bazel target - especially for new code)

Reply:
Done

Reply:
Done